sdk/src/gql/index.ts

```diff
@@ -1,13 +1,12 @@
 import { inject, injectable } from 'inversify';
 import IGqlClient from '@akashaproject/sdk-typings/lib/interfaces/gql';
 import { ServiceCallResult } from '@akashaproject/sdk-typings/lib/interfaces';
-import { ApolloLink, HttpLink, gql, GraphQLRequest } from '@apollo/client';
+import { gql, GraphQLRequest, HttpLink } from '@apollo/client';
 
 import hash from 'object-hash';
 import { TYPES } from '@akashaproject/sdk-typings';
 import Stash, { IQuickLRU } from '../stash';
-import { createObservableStreamGql, createObservableValue, toPromise } from '../helpers/observable';
-import { tap } from 'rxjs/operators';
+import { createObservableStreamGql, createObservableValue } from '../helpers/observable';
```

Comment:
Codacy has a fix for the issue: [Insert `';⏎import·Stash,·{·IQuickLRU·}·from·'../stash`](https://app.codacy.com/gh/AKASHAorg/akasha-world-framework/pullRequest?prid=8098751)
```suggestion
import { createObservableStreamGql, createObservableValue } from '../helpers/observable';
```